CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
 ### Added
+* [508](https://trello.com/c/LohvvmPO/508-add-youtube-to-honestybox-measurement) Add youtube measurement
```

Comment:
@jamesaj23 I didn't pick this up at review but note that you hadn't done a `git rebase master` so your `CHANGELOG.md` was out of date. The two lines below this should have been underneath the `1.0.7` release.

No biggie, just FYI remember to rebase your feature branch, particularly if `master` has updated since you last worked on it.